protocol/messenger_chats.go

```diff
@@ -46,10 +46,19 @@ func (m *Messenger) Chats() []*Chat {
 	return chats
 }
 
-func (m *Messenger) ChatsPreview() []*ChatPreview {
+// ChatsPreview returns a list of chat previews.
```

Comment:
It would be easier to understand and less prone to bugs if we didn't use a tri-state boolean. While integrating with this PR https://github.com/status-im/status-mobile/compare/ilmotta/use-optimized-filter-initialization?expand=1 I had to come back to this comment multiple times to make sure I wasn't misunderstanding anything. A better solution here I think is to use an enum, which would be self-explanatory and wouldn't require a comment.

In the client layer I had to create 3 constants to describe what the tri-state flag means:

```clojure
(def ^:const chat-preview-type-any nil)
(def ^:const chat-preview-type-community true)
(def ^:const chat-preview-type-non-community false)
```

Reply:
Will apply, thanks